src/test/java/com/amihaiemil/zold/RtWalletTestCase.java

```diff
@@ -199,4 +199,38 @@ public void findsWalletTransactions() throws Exception {
             Matchers.equalTo(payload)
         );
     }
+
+    /**
+     * Checks whether the {@link RtWallet#pay(String, String, String, String)}
+     * invocation succeeds.
+     * @throws Exception If an exception is thrown
+     */
+    @Test
+    public void pays() throws Exception  {
+        final ClassicHttpResponse response = new Response(
+                HttpStatus.SC_OK,
+                ContentType.TEXT_PLAIN,
+                "12345"
+        );
+        final HttpClient httpClient = new MockHttpClient(
+                new AssertRequest(
+                        response,
+                        new Condition(
```

Comment:
@ammaratef45 , could you please add another condition for checking the request payload?

Reply:
@denis-konovalyenko the pay method is void and payload isn't visible, can you explain how I can test it? Thanks

Reply:
@ammaratef45 , try to use `com.amihaiemil.zold.PayloadOf#PayloadOf` wiith `com.amihaiemil.zold.mock.Condition'.

Reply:
@ammaratef45 , I have made a related commit in my branch [here](https://github.com/denis-konovalyenko/zold-java-client/tree/61). Could you please take a look when have time?

Reply:
@ammaratef45  ping :)

Reply:
@denis-konovalyenko I forgot about this PR :)
I've merged your branch, Thanks for help

Reply:
@ammaratef45 , thanks!